backend/internal/handlers/classrooms/assignments/works/service.go

```diff
@@ -13,3 +18,21 @@ type WorkService struct {
 func NewWorkService(store storage.Storage, githubappclient github.GitHubAppClient) *WorkService {
 	return &WorkService{store: store, githubappclient: githubappclient}
 }
+
+// Helper function for getting a student work by ID
+func getWork(s *WorkService, c *fiber.Ctx) (*models.PaginatedStudentWorkWithContributors, error) {
```

Comment:
I feel like this should be defined in the same file as all of the other works functionality.

Reply:
ahh ok I missed that, then yah thats cool